applications/launchpad/gui-react/src/store/settings/selectors.ts

```diff
@@ -17,6 +17,7 @@ export const selectServiceSettings = (state: RootState) => ({
   moneroUsername: state.mining.merged.authentication?.username || '',
   moneroPassword: state.mining.merged.authentication?.password || '',
   rootFolder: state.baseNode.rootFolder,
+  walletPassword: state.settings.serviceSettings.parole,
```

Comment:
Possibly, no longer needed. You may need to rebase first.

Reply:
Checked and removed.